iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/ReplaceTeePointByVoltageLevelOnLine.java

```diff
@@ -335,7 +335,7 @@ public void apply(Network network, boolean throwException,
         removedLineReport(reporter, line1ZId);
         lineZ2.remove();
         removedLineReport(reporter, lineZ2Id);
-        lineZP.remove();
+        lineZP.remove(true);
```

Comment:
From what I have understand, this feature is quite buggy and will be removed by @flo-dup . Maybe it is better to wait until the correction is made (it will call another modification)

Reply:
indeed! see PR #2340

Reply:
You can now use `RemoveFeederBay`!